tlslite/keyexchange.py

```diff
@@ -292,6 +297,38 @@ def makeClientKeyExchange(self):
         cke.createDH(self.dh_Yc)
         return cke
 
+# the DHE_RSA part comes from IETF ciphersuite names, we want to keep it
+#pylint: disable = invalid-name
+class DHE_RSAKeyExchange(ADHKeyExchange):
+    """
+    Handling of ephemeral Diffe-Hellman Key exchange
+
+    NOT stable API, do NOT use
+    """
+
+    def __init__(self, cipherSuite, clientHello, serverHello, privateKey):
+        super(DHE_RSAKeyExchange, self).__init__(cipherSuite, clientHello,
+                                                 serverHello, privateKey)
+#pylint: enable = invalid-name
+        self.dh_Xs = None
+        self.dh_Yc = None
```

Comment:
why you're re-setting values of those fields? doesn't ADHKeyExchange already do that?